hoomd/md/pytest/conftest.py

```diff
@@ -0,0 +1,533 @@
+import hoomd
+import pytest
+import numpy as np
+from copy import deepcopy
+import itertools
+
+np.random.seed(0)
```

Comment:
Remove this. The main `conftest.py` file sets the numpy random seed to the same seed at the start of each test function.

Reply:
Resolved 92252586f